streetcomplete-dark-style.yaml

```diff
@@ -13,32 +13,29 @@ import:
 #    - layers/roads.yaml
 
 global:
-    text_stroke_color: '#2e2e48'
-    text_fill_color: '#ccf'
-    text_water_color: '#ccf'
-
-    railway_color: '#96c'
-    road_color: '#559'
-    road_outline_color: '#99f'
-    highway_color: '#669'
-    highway_outline_color: global.road_outline_color
-    path_color: global.road_color
-    path_outline_color: '#547'
-    square_color: global.road_color
-
-    earth_color: '#2e2e48'
-    water_color: '#002'
-    green_color: '#363054'
-    forest_color: '#403962'
-    town_color: '#3d364e'
+    text_stroke_color: [0.184, 0.180, 0.165, 1.00]
+    text_fill_color: '#ccc'
+    text_water_color: '#ccc'
+
+    railway_color: [0.396, 0.396, 0.502, 1.00]
+    road_color: [0.31, 0.31, 0.31, 1.00]
+    road_outline_color: [0.184, 0.180, 0.169, 1.00]
+    highway_color: [0.533, 0.424, 0.424, 1.00]
+    path_outline_color: [0.259, 0.247, 0.220, 1.00]
+
+    earth_color: [0.055, 0.047, 0.043, 1.00]
+    water_color: [0.106, 0.122, 0.243, 1.00]
+    green_color: [0.137, 0.145, 0.125, 1.00]
+    forest_color: [0.067, 0.110, 0.078, 1.00]
+    town_color: [0.087, 0.090, 0.090, 1.00]
```

Comment:
(BTW it shows more noticeable on real phone; screenshots on desktop do not do it justice).

Reply:
Completely black for landuses? But then, there is no diference in color between the "background" and e.g.  `landuse=residential`.

No, I think the issue here is that the contrast between the buildings and the buildings is too low in the left picture. In general, I think I overdid it a bit with the decrease in brightness.

After all, even in "dark mode" on the desktop PC, you have white text on black background. Not dark grey text on black background. 

And as people want to use the dark theme also in the sun (@Helium314 , ...), it would make sense if it had a little more contrast, too.

So maybe I should try out:
- lighter streets and names
- do something about the contrast between buildings and background
- slightly brighter landuses etc